dxr/app.py

```diff
@@ -249,6 +251,23 @@ def raw(tree, path):
     return send_file(data_file, mimetype=guess_type(path)[0])
 
 
+@dxr_blueprint.route('/<tree>/rawrev/<revision>/<path:path>')
```

Comment:
So far we've managed to keep underscores out of the URL, which is a nice thing, I think (esp. when we're already using dashes). I haven't had a chance to look through your PR in detail yet, but I vaguely and nebulously wonder if we really need this extra endpoint. I suppose it's to distinguish a binary's containing page from its raw data?